javascript/index.js

```diff
@@ -125,30 +125,113 @@ window.onload = function () {
     },
   })
 
-  //TODO - Put logic to get all forms work
-  var form = document.getElementById('form-1')
+  var form1 = document.getElementById('form-1')
```

Comment:
¿Qué opinas si estas vars quedan de la siguiente forma?
```
var form1 = document.getElementById('form-1'),
    form2 = document.getElementById('form-2'),
    form3 = document.getElementById('form-3'),
    form4 = document.getElementById('form-4'),
    form5 = document.getElementById('form-5');
```

Reply:
https://stackoverflow.com/questions/694102/declaring-multiple-variables-in-javascript
por que seria mejor tenerlo de esa forma ? 🤔  @JoValo

Reply:
Gracias por el articulo :)